src/lib/y2storage/proposal/autoinst_drives_map.rb

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
+#
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/proposal/skip_list"
+
+module Y2Storage
+  module Proposal
+    # Utility class to map disk names to the corresponding AutoYaST <drive>
+    # specification that will be applied to that disk.
+    class AutoinstDrivesMap
+      extend Forwardable
+
+      # @!method each_pair
+      #   Calls block once por each disk that contains the AutoYaST specification
+      #   passing as arguments the disk name and the specification itself.
```

Comment:
link to specification would be great, especially if someone want to reuse it for something different then only autoyast.